docs/nodes.md

```diff
@@ -115,6 +115,10 @@ Even so, running applications should continue to be available.
 If you add multiple ectd nodes with one run, you might want to append `-e etcd_retries=10` to increase the amount of retries between each ectd node join.
 Otherwise the etcd cluster might still be processing the first join and fail on subsequent nodes. `etcd_retries=10` might work to join 3 new nodes.
 
+### 4) Add the new node to apiserver config
```

Comment:
This section is for `## Adding an etcd node` and the previous number is 1.
So here should be 2 instead of 4.

Reply:
fixed, thank you!